runtime/compiler/env/PersistentCHTable.hpp

```diff
@@ -46,6 +46,24 @@ namespace TR { class ResolvedMethodSymbol; }
 
 class TR_PersistentCHTable
    {
+   private:
+   /**
+    * @brief Enum to denote the state of the CH Table:
+    *
+    *        active: Data in the CH Table can be used and updated.
+    *        activating: CH Table is in the process of being activated;
+    *                    this state is used when the CH Table is
+    *                    being populated in the restore run.
+    *        reset: CH Table is not active.
```

Comment:
Should `reset` be first to match the ordering in the enum?

Reply:
Done, see https://github.com/eclipse/openj9/compare/0efdded6403247d28fca53431cb1a2abe4487fdf..043c3c65ba6a7889d13137fccdf5885a384e3e2e